src/celeritas/inp/MucfPhysicsData.hh

```diff
@@ -0,0 +1,117 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/inp/MucfPhysicsData.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Macros.hh"
+#include "corecel/inp/Grid.hh"
+
+#include "MucfPhysics.hh"
+
+namespace celeritas
+{
+namespace inp
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Static muon energy CDF data for muon-catalyzed fusion.
+ *
+ * \todo This data may be loaded from an external file in the future.
+ */
+static Grid mucf_muon_energy_cdf()
```

Comment:
These should probably be inlined directly into `MucfPhysics::from_default`. At any rate, they should live inside that cc file.